pkg/config/process.go

```diff
@@ -0,0 +1,79 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package config
+
+import (
+	"os"
+	"strings"
+	"time"
+)
+
+const (
+	// DefaultGRPCConnectionTimeoutSecs sets the default value for timeout when connecting to the agent
+	DefaultGRPCConnectionTimeoutSecs = 60
+)
+
+// procBindEnvAndSetDefault is a helper function that generates both "DD_PROCESS_CONFIG_" and "DD_PROCESS_AGENT_" prefixes from a key.
+// We need this helper function because the standard BindEnvAndSetDefault can only generate one prefix from a key.
+// Additionally, procEnvAndSetDefault allows us to additively provide environment variables, which BindEnvAndSetDefault does not.
```

Comment:
Is this comment line still valid?

Reply:
I think line 21 is not valid anymore. Great catch!